pkg/webui/console/views/application-general-settings/index.js

```diff
@@ -171,6 +202,14 @@ export default class ApplicationGeneralSettings extends React.Component {
                 name="description"
                 component={Input}
               />
+              <Form.Field
+                name="attributes"
+                title={sharedMessages.attributes}
+                keyPlaceholder={sharedMessages.attribute}
+                valuePlaceholder={sharedMessages.value}
```

Comment:
Please also add a description for the attribute field. Something like `Attributes can be used to set arbitrary information about the application, to be used by scripts, or simply for your own organization.` cc @htdvisser

Maybe it would be good to add a section in our documentation about the use of attributes as well.